zeth_contracts/contracts/BW6_761_test.sol

```diff
@@ -9,18 +9,18 @@ contract BW6_761_test
     // In many cases, these numbers must be used as literals in the assembly
     // code.
 
-    uint256 constant scalarWords = 2;
-    uint256 constant scalarBytes = scalarWords * 32; // 64 (0x40)
-    uint256 constant coordWords = 3;
-    uint256 constant coordBytes = 3 * 32; // 96 (0x60)
-    uint256 constant pointWords = 2 * coordWords;
-    uint256 constant pointBytes = pointWords * 32; // 192 (0xc0)
+    uint256 internal constant SCALAR_WORDS = 2;
+    uint256 internal constant SCALAR_BYTES = SCALAR_WORDS * 32; // 64 (0x40)
+    uint256 internal constant COORD_WORDS = 3;
+    uint256 internal constant COORD_BYTES = 3 * 32; // 96 (0x60)
+    uint256 internal constant POINT_WORDS = 2 * COORD_WORDS;
+    uint256 internal constant POINT_BYTES = POINT_WORDS * 32; // 192 (0xc0)
```

Comment:
Unused?

Reply:
Btw all these can be made `private` I think. Would be good to tighten the scopes.